data/015/5010/en_US.json

```diff
@@ -3,11 +3,11 @@
 		"5010": {
 			"name": "Unknown",
 			"message": "You cannot post to Miiverse from\nthis system.\n\nFor details, please start Miiverse",
-			"short_description": "Your console has been banned from Juxtaposition",
-			"long_description": "Unknown cause.",
-			"short_solution": "Don't do whatever you did to get banned",
-			"long_solution": "Unknown solution.",
+			"short_description": "Your console has been restricted from Juxtaposition",
+			"long_description": "Your console has been restricted from Juxtaposition.\n\nThis typically occurs because of a violation of the Code of Conduct, or other offenses on the Network or the Discord server.",
```

Comment:
```suggestion
			"long_description": "Your console has been restricted on Juxtaposition.\n\nThis typically occurs because of a violation of the Code of Conduct, or other offenses on the Network or the Discord server.",
```